src/Value/LineName.php

```diff
@@ -12,7 +12,7 @@
 class LineName extends ValueList
 {
     /**
-     * @param array<int, Value|string> $components
+     * @param array<Value|string> $components
```

Comment:
Won't this weaken the type specification, also allowing strings as array keys?

Reply:
Yes, it does. As far as I can see, the array keys are not used, and hence it doesn't matter if string or int array keys are passed in.

Reply:
OK.